docs/specs/src/hwp_1_intro.md

```diff
@@ -0,0 +1,212 @@
+Holochain
+
+Distributed Coordination by Scaled Consent not Global Consensus
+
+Abstract
+========
+
+In this paper we present a frame and a specification for a generalized  
+system for large scale distributed coordination that does not rely on
+global consensus. We start with some axioms; we proceed to describe the
+consequences of these axioms, looking at the problem through the lens of
+scaling consent; we present an informal description of the system; we
+provide a more formal outline of the necessary integrity guarantees and
+system components needed to deliver those guarantees; and finally we
+conclude with a high-level yet sufficiently detailed technical
+specification of what's need to implement such a system.
+
+Introduction
+============
+
+Preamble
+--------
+
+The first version of the [[Holochain white
+paper]{.underline}](https://github.com/holochain/holochain-proto/blob/whitepaper/holochain.pdf)
+took on a very formal approach to describing generalized distributed
+computation schemes and carefully compared the approach taken by
+Blockchain based systems with the approach taken by Holochain as well as
+providing formal reasoning for the benefits of Holochain's agent centric
+approach.
+
+In this second version, we describe Holochain on its own terms by
+providing the highest level view of our approach and our starting
+axioms. At the next level of detail we provide an informal system
+description using the metaphor of games. Then we dive into deeper detail
+with a formal system description including the specific context and
+assumptions we come from, the integrity guarantees Holochain offers, and
+a formal state model and discussion of security and safety concerns.
+Finally at the most detailed level we provide an implementation
+specification in Appendix A.
+
+Axioms
+------
+
+We begin by stating our axioms[^1] regarding the nature of coordination:
+
+1.  **Coordination arises from agents** starting from the same
+  ground-rules and acting as soon as each party in any interaction
+  can confirm that each other\'s actions confirm that actions or
+  interactions conform to those ground rules. (Thus, in our frame,
+  coordination looks like alignment heading in the same direction,
+  rather than agreement proceeding in lockstep.)
+
+2.  **Coordination is grammatic**, in that it comes from embodying a
+  geometry that removes uncertainty, and embodying an ability to
+  compose different coordinative subsystems that have different
+  ground-rules.
```

Comment:
Why different ground rules? The subsystems share the ground rules, as stated further down for social coherence (authority), that all agents know the ground rules.

Reply:
Each DNA is a set of ground-rules.  The overall system embodies being able to compose them.